cmd/up/up.go

```diff
@@ -226,10 +231,6 @@ func (up *upContext) start(autoDeploy, build bool) error {
 		return fmt.Errorf("failed to load your local Kubeconfig: %q context not found in %q", up.Dev.Context, kubecfg)
 	}
 
-	if up.Dev.Namespace == "" {
```

Comment:
We can probably remove similar lines from other commands, for example here:
https://github.com/okteto/okteto/blob/master/cmd/down.go#L99
if we move the namespace initialization iiside `dev.LoadContext` instead of `loadDevOverrides`

Reply:
The problem of moving the logic to that function is that it creates a cycle import because of client -> analytics -> config -> model -> client. I have to check how can this logic be implemented without changing the main structure of the project.